### PR TITLE
Add a link of README.md by Korean

### PR DESCRIPTION
**Pull request by LeeHyungGeun**
This commit is just added a link of the translation which is translated as Korean in leehyunggeun/react-beautiful-dnd-kr

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -866,6 +866,10 @@ This library supports the standard [Atlassian supported browsers](https://conflu
 
 Currently mobile is not supported. However, there are plans to add touch support in the future
 
+## Translation
```
Comment (alexreardon) [inline README.md]: Community supported translations

File: README.md [hunk 2]
```diff
@@ -866,6 +866,10 @@ This library supports the standard [Atlassian supported browsers](https://conflu
...
+This library is also available in other languages:
```
Comment (alexreardon) [inline README.md]: The documentation for this library is also available in other languages:

File: README.md [hunk 3]
```diff
@@ -866,6 +866,10 @@ This library supports the standard [Atlassian supported browsers](https://conflu
...
+- ![kr](https://raw.githubusercontent.com/gosquared/flags/master/flags/flags/shiny/24/South-Korea.png) **Korean**: [leehyunggeun/react-beautiful-dnd](https://github.com/LeeHyungGeun/react-beautiful-dnd-kr)
```
Comment (alexreardon) [inline README.md]: Could the fork of the repo just include the `README.md` rather than the source also? What do you think?

**Comment by alexreardon** [comment]
@LeeHyungGeun this is amazing. Thank you for doing this! ❤️

File: README.md (hunk 3 above)
Comment (alexreardon) [inline README.md]: Actually never mind about that - it would make updating translations harder

Comment (LeeHyungGeun) [inline README.md]: Do you mean that the fork, of the repo, what I forked from the original repo just include the `README.md`?

File: README.md (hunk 1 above)
Comment (LeeHyungGeun) [inline README.md]: I fixed.

File: README.md (hunk 2 above)
Comment (LeeHyungGeun) [inline README.md]: I fixed.

**Pull request merged**